CHANGES.txt

```diff
@@ -1,4 +1,5 @@
-﻿Current
+Current
+Fixed: GITHUB-1302: while set parallel to classes the program may occur the exception(Jianhua Li)
```

Comment:
Rephrase: "When 'parallel' is set to 'classes', `ConcurrentModificationException` can be thrown.